cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -369,34 +366,36 @@ loop:
 }
 
 // sendCommitteeTx creates transaction from script, signs it by committee nodes and sends it to RPC.
-// If tryGroup is false, global scope is used for the signer (useful when
-// working with native contracts).
-func (c *initializeContext) sendCommitteeTx(script []byte, tryGroup bool) error {
-	return c.sendMultiTx(script, tryGroup, false)
+// If fancyScope is true, universal NeoFS scope is used for the signer (useful when
```

Comment:
is it still "useful when working with native contracts"?

Reply:
Not really, fixed.